### PR TITLE
Add jackson-annotations dep

### PR DESCRIPTION
**Pull request by davidxia**

### CODE REVIEW

File: common/pom.xml [hunk 1]
```diff
@@ -26,6 +26,11 @@
       <artifactId>jackson-databind</artifactId>
       <version>2.4.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+      <version>2.4.0</version>
```
Comment (marcusb) [inline common/pom.xml]: the version should probably match that of jackson-databind

Comment (davidxia) [inline common/pom.xml]: `mvn dependency:tree` gave me 2.4.0 for some reason. So I left that as is. We can use 2.4.1 though.

```
[INFO] ------------------------------------------------------------------------
[INFO] Building Spotify Docgenerator Common Classes 0.0.2-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ common ---
[INFO] com.spotify.docgenerator:common:jar:0.0.2-SNAPSHOT
[INFO] +- javax.ws.rs:jsr311-api:jar:1.1.1:compile
[INFO] +- com.google.guava:guava:jar:17.0:compile
[INFO] +- com.fasterxml.jackson.core:jackson-databind:jar:2.4.1:compile
[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.4.0:compile
[INFO] |  \- com.fasterxml.jackson.core:jackson-core:jar:2.4.1:compile
[INFO] \- com.intellij:annotations:jar:12.0:provided
```

**Comment by drewcsillag** [comment]
:+1:

**Comment by davidxia** [comment]
@drewcsillag How do I release this?

**Pull request merged**